### PR TITLE
Se 301.python preview parity cov runner

### PR DESCRIPTION
**Pull request by slacki123**
# Pull Request Checklist

- [x] Read the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: sdk/features/main.py [hunk 1]
```diff
@@ -1,37 +1,10 @@
-import argparse
-import os
 import sys
 
-from features.feature_extractor import extract_all_features_from_package
-from features.feature_file_writer import write_features_to_file
-from features.get_project_root import get_project_root
+from lusidfeature.entrypoint import extract_features_to_file
```
Comment (rizwansaeed) [inline sdk/features/main.py:3]: having seen it now, we should change this to `lusidfeature.reporter` 

sorry!

Comment (slacki123) [reply sdk/features/main.py]: updated!

**Review by rizwansaeed — APPROVED** [review]

File: sdk/requirements.txt [hunk 2]
```diff
@@ -4,4 +4,5 @@ python-dateutil >= 2.5.3
 setuptools >= 21.0.0
 urllib3 >= 1.15.1
 pytz >= 2019.1
-requests >= 2.21.0
\ No newline at end of file
+requests >= 2.21.0
+lusidfeature >= 0.1.3
```
Comment (rizwansaeed) [inline sdk/requirements.txt:8]: we don't need the version here, we will most likely always need the latest

File: sdk/requirements.dev.txt [hunk 3]
```diff
@@ -1 +1,2 @@
-parameterized==0.7.*
\ No newline at end of file
+parameterized==0.7.*
+lusidfeature >= 0.1.3
```
Comment (rizwansaeed) [inline sdk/requirements.dev.txt:2]: remove lusidfeature from here and add the following as the first line

`-r requirements.txt`

**Review by rizwansaeed — CHANGES_REQUESTED** [review]

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**